src/main/java/tickbot/task/Task.java

```diff
@@ -0,0 +1,73 @@
+package tickbot.task;
+
+import java.time.LocalDate;
+
+/**
+ * The class to represent an abstract task.
+ */
+public abstract class Task {
+    private final String content;
+    private final LocalDate time;
```

Comment:
Partly because I do not want to repeat the code in toString() for a task. If I do not put time in Task, I have to write the same code twice in Deadline and Event.
Another reason is that I store the task list in CSV format, which is essentially a table recording task type, contents, completed or not as well as time. So I need a time (even if it could be null) for all types of tasks to fill in the data file.

Reply:
To my knowledge, `java.time.LocalDate` does not encapsulate time specifically, only date. The use of the variable name `LocalDate time` in this case might be misleading. May I clarify what form of date or time tracking did you intend to implement?